README.md

```diff
@@ -582,6 +582,49 @@ Then, import it:
 Generate a nix attribute set from the opam file. This is just a Nix
 representation of the JSON produced by `opam2json`.
 
+### Monorepo functions
+
+#### `queryToMonorepo`
+
+```
+{ repos = ?[Repository]
+; resolveArgs = ?ResolveArgs
+; filterPkgs ?[ ] }
+→ Query
+→ Scope
+```
+
+Similar to `queryToScope`, but creates a attribute set (instead of a
+scope) with package names mapping to sources for replicating the
+[`opam monorepo`](https://github.com/tarides/opam-monorepo) workflow.
+
+The `filterPkgs` argument gives a list of package names to filter from
+the resulting attribute set, rather than removing them based on their
+opam `dev-repo` name.
+
+#### `buildOpamMonorepo'`
+
+```
+{ repos = ?[Repository]
+; resolveArgs = ?ResolveArgs
+; pinDepends = ?Bool
+; recursive = ?Bool
+; extraFilterPkgs ?[ ] }
+→ project: Path
+→ Query
+→ Scope
```

Comment:
This doesn't quite return a `Scope`, it returns an attrset of sources, doesn't it?